compose-app/src/main/java/com/mapbox/maps/compose/ui/theme/Theme.kt

```diff
@@ -0,0 +1,38 @@
+package com.mapbox.maps.compose.ui.theme
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.darkColors
+import androidx.compose.material.lightColors
+import androidx.compose.runtime.Composable
+
+private val DarkColorPalette = darkColors(
+  primary = Purple200,
+  primaryVariant = Purple700,
+  secondary = Teal200
+)
+
+private val LightColorPalette = lightColors(
+  primary = Purple500,
+  primaryVariant = Purple700,
+  secondary = Teal200
+)
+
+@Composable
+fun TestMapboxMapComposeTheme(
```

Comment:
do we need if?

and maybe rename
```suggestion
fun MapboxMapsComposeTheme(
```